doc/source/_static/css/custom.css

```diff
@@ -3,6 +3,7 @@
 * for more information. `important!` is needed below to override
 * dark/light theme specific values, which normally take precedence over the PST defaults.
 * */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;900&family=Poppins&family=Roboto:wght@400;500;700&display=swap');
```

Comment:
why is this required now? and this not only import `Poppins` but also `Roboto`, which is not used in this css file? is it that important to use the `Poppins` font? It will make page loading slightly slower, right?

Reply:
> is it that important to use the Poppins font? It will make page loading slightly slower, right?

So I'm in agreement about this - IMO we should not be loading a new font here for a single element either, nor do I think we should be introducing new colors rather than using existing Ray branding. But these designs were been passed to me from marketing, so I didn't think I had wiggle room to change this.

---

EDIT: After talking with Angelina about this and some other design considerations, it seems like Poppins is not needed.